tests/test_characteristicfunctions.py

```diff
@@ -32,8 +32,7 @@
 
 
 def testSphere(sas_available):
-    if not sas_available:
-        pytest.skip("sas package not available")
+    pytest.skip("calculate_ER() not available")
```

Comment:
this is the wrong error message.  I think the ER function is not the issue.   Just say "sas characteristic functions not currently working.  Remove skip when our code is refactored to use the latest sasview API" or sthg like that.

Also, let's keep the original skip but commented out.  We will need it again when we implement the new code.